klasses/api_test.py

```diff
@@ -24,7 +25,7 @@
 
 
 @pytest.fixture()
-def setup_test_data():
+def setup_test_data(mocker):
```

Comment:
i don't know what benefit we get from the naming similarity with unittest/xUnit. if we keep this, all fixtures we define and use will be nouns while this one is a verb. i think it's more beneficial to think about fixtures the way pytest does and follow their conventions ([every fixture in the docs has a noun name](https://docs.pytest.org/en/latest/fixture.html)) than to keep a vague reference to our old test framework. anyway i'll leave this up to you. not something to hold up the PR for